src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -215,8 +215,8 @@ class LazyClassLoaderHelper
                                         llvm::StringRef fieldName, llvm::StringRef fieldType)
     {
         return returnConstantForClassObject(
-            builder, "L" + className + ";",
-            [=](const ClassObject* classObject)
+            builder, "L" + className + ";", fieldName + "" + fieldType,
```

Comment:
Having a `+` with an empty string is a bit redundant. Either just write `fieldName + fieldType` or use the same delimiter as below (`;`)